macosx/Compatibility.h

```diff
@@ -0,0 +1,52 @@
+//
+//  Compatibility.h
+//  Transmission
+//
+//  Created by Antoine Coeur on 31/03/2022.
+//  Copyright © 2022 The Transmission Project. All rights reserved.
```

Comment:
Header here and in .m file differs from others. Should this include license information? CC @ckerr